pkg/agent/openflow/pipeline.go

```diff
@@ -959,8 +960,13 @@ func (c *client) bridgeAndUplinkFlows(uplinkOfport uint32, bridgeLocalPort uint3
 		ctStateNext = endpointDNATTable
 	}
 	flows := []binding.Flow{
-		// Forward the packet to conntrackTable if it enters the OVS pipeline from the uplink interface.
 		c.pipeline[ClassifierTable].BuildFlow(priorityNormal).
+			MatchInPort(uplinkOfport).
+			Action().GotoTable(uplinkTable).
+			Cookie(c.cookieAllocator.Request(category).Raw()).
+			Done(),
+		// Forward the packet to conntrackTable if it enters the OVS pipeline from the uplink interface.
+		c.pipeline[uplinkTable].BuildFlow(priorityNormal).
```

Comment:
I guess only the traffic from the uplink interface is entering theuplinkTable, then maybe we don't need the "in_port" match in this table.

Reply:
Done.